### ISSUE TITLE
Tables within Notes: alternate colour?

### ISSUE THREAD

**Issue by Gadget2020** [naturalcrit/homebrewery#479]
When you put a table into a note (with '>'): since the background colour of the note is the same as the alternate highlight colour of the table, the 'spaces' in the table become the same colour as the highlight... so you loose the definition. 
Request: Can the highlight colour be changed to white when it's on a note?

**Comment by Rae2che5** [comment]
You could try adding the following to the start of your brew:

```
<style>
.phb blockquote table tbody tr:nth-child(odd) { background-color: #ffffff; }
</style>
```

**Comment by Sansd20** [comment]
@Rae2che5 this works for the whole sheet. I have several tables and I am only wanting one table to have a different background color (transparent). Is there a way to do this?  I tried adding `style='background-color : transparent'` to the div container with no luck.

**Comment by Rae2che5** [comment]
You'd need to somehow identify the table that you want to affect. I suggested a mechanism for doing that in #348. For example, you could add a line saying `###### transparent-bg` just before the table, and use the following CSS:

```html
<style>
/* Hide the heading. */
h6 { display: none; }
/* Affect table immediately following the heading. */
h6[id|='transparent-bg'] table tbody tr:nth-child(odd) { background-color: transparent; }
</style>
```

Note that this won't do what you want in a note (i.e. in a `>`-marked blockquote), because making the rows transparent merely allows the note's background to be visible. I'm not quite sure what effect you're trying to achieve -- could you elaborate?

**Comment by Sansd20** [comment]
I should mention that I am not working inside a note on this one.

so, [here](http://homebrewery.naturalcrit.com/share/By1ppsLo-) is the sheet I am working on. I have sidestep the issue by using a descriptive box, and not a table.

The 'prepared spells' part I want to put in a table, and I want that table to be shaded. I am also wanting the to keep the table in the classTable as they are, transparent/white.

**Comment by Rae2che5** [comment]
You mean like [here](http://homebrewery.naturalcrit.com/share/SJmTxSLLzG)?

**Comment by Sansd20** [comment]
@Rae2che5 Yes! The tastiest of cookies!

**Comment by calculuschild** [comment]
Closing because this appears to be solved.

**Comment by calculuschild** [comment]
Aha. @HeruAsatru found one in the DMG

![image](https://user-images.githubusercontent.com/5878534/48713949-95e75400-ebdf-11e8-844f-8a30277fc1ec.png)

**Comment by HeruAsatru** [comment]
In notes, we could have it alternate between white and "transparent" (both the note background and the table background are #e0e5c1).

In descriptive text boxes, we could have it alterneate between that green background (#e0e5c1) and "transparent", meaning white.

**Comment by Gazook89** [comment]
Just some perusing, and found a table in the 5e PHB inside a note, which does *not* alternate colors....:

<img width="381" alt="image" src="https://user-images.githubusercontent.com/58999374/130308038-7ae70e58-3090-4069-9f90-c930428834ac.png">

pg. 310

**Comment by Gazook89** [comment]
So PHB and DMG seem to handle this differently, PHB doing no row backgrounds and DMG using lighter tones.   I worked up a good change to the PHB.less that was going to match the DMG, and was going to submit as a PR, but am now thinking the PHB.less stylesheet should stay true to the PHB itself. 

This might mean creating a separate DMG stylesheet, tied to the 'system' metadata?

**Comment by 5e-Cleric** [comment]
This will be fixed for the DMG theme in #2796, and as Gazook stated two years ago, it will not be fixed in the PHB. Closing this issue as completed for now.
